pytos/securetrack/xml_objects/rest/device.py

```diff
@@ -262,11 +266,17 @@ def from_xml_node(cls, xml_node):
         revision_id = get_xml_int_value(xml_node, xml_tags.Elements.REVISIONID)
         revision_time = get_xml_text_value(xml_node, xml_tags.Elements.TIME)
         ready = get_xml_text_value(xml_node, xml_tags.Elements.READY)
+        ticket = []
+        for ticket_node in xml_node.iter(tag=xml_tags.Elements.TICKET):
+               ticket_obj = Device_Revision_Ticket.from_xml_node(ticket_node)
+               ticket.append(ticket_obj)	
+
+
         return cls(action, num_id, admin, auditLog, authorizationStatus, revision_date, revision_time, gui_client,
-                   revision_id, modules_and_policy, policy_package, ready)
+                   revision_id, modules_and_policy, policy_package, ready,ticket)
 
     def is_authorized(self):
-        return self.authorizationStatus in Device_Revision.AUTHORIZED_STATUSES
+        return self.authorizationStatus == Device_Revision.AUTHORIZED_STATUS
```

Comment:
Once `"authorized"` is added to `AUTHORIZED_STATUSES` tuple this line shouldn't be changed